gui-apps/kanshi/kanshi-9999.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 2019-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson
+
+DESCRIPTION="dynamic display configuration (autorandr for wayland)"
+HOMEPAGE="https://github.com/emersion/kanshi"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/emersion/kanshi"
+else
+	SRC_URI="https://github.com/emersion/kanshi/releases/download/v${PV}/${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm64 ~x86"
+fi
+
+LICENSE="MIT"
+SLOT="0"
+IUSE="doc"
+
+RDEPEND="
+	dev-libs/wayland
+"
+BDEPEND="
+	${RDEPEND}
+	doc? (
+		app-text/scdoc
```

Comment:
This should depend on the live ebuild for app-text/scdoc. See gui-wm/sway for an example.

Reply:
I've added the sway version.